package.json

```diff
@@ -4,11 +4,13 @@
   "description": "An anonymous chat site",
   "main": "index.js",
   "scripts": {
-    "start": "node app.js"
+    "start": "node app.js",
+    "test:watch": "mocha --watch ./*.test.js",
+    "test": "mocha ./*.test.js"
   },
   "engines": {
     "npm": "3.10.10",
-    "node": "6.9.4"
+    "node": "12.19.0"
```

Comment:
Do not update the dependencies, because this crashed the app on heroku.

Reply:
mocha@8.2.0 expect a version of node >= 10.12.0. Any suggestion?

Reply:
Okay, leave it, I just tested this version of app. It is working fine.